File: backend/dtos/companyDto.py
class CompanyDto:
    def __init__(self, id: int, name: str, phoneNumber: str, email: str, website: str, ownerName: str, address: any, rating: int):
        self.id = id
        self.name = name
        self.phoneNumber = phoneNumber
        self.email = email
        self.website = website
        self.ownerName = ownerName
        self.address = address
        self.rating = rating

File: frontend/src/lib/dtos/companyDto.ts
import type { AddressDto } from "./adresDto";

export type CompanyDto = {
    id: number;
    name: string;
    phone_number?: string;
    email?: string;
    website_url?: string;
    owner_name: string;
    rating: number;
    address: AddressDto;
}
